util/src/main/java/tech/pegasys/teku/util/config/NetworkDefinition.java

```diff
@@ -45,6 +45,15 @@
                   .eth1Endpoint("https://goerli.prylabs.net")
                   .startupTimeoutSeconds(120)
                   .build())
+          .put(
+              "altona",
+              builder()
+                  .constants("altona")
+                  .snappyCompressionEnabled(true)
+                  .eth1DepositContractAddress("0x16e82D77882A663454Ef92806b7DeCa1D394810f")
+                  .eth1Endpoint("https://goerli.prylabs.net")
```

Comment:
Not sure if we want to have the Eth1 endpoint point to prylabs.net, but I kept it for now since I couldn't figure out a better solution.

Reply:
Just remove it.  We don't normally provide a default ETH1 endpoint - I just included the pylabs one for Onyx since that's what it was put up for.